changelog 2.0.0-beta.3 (#4181)
mapbox/mapbox-navigation-android: CHANGELOG.md

Mapbox welcomes participation and contributions from everyone.

## Mapbox Navigation SDK 2.0.0-beta.3 - March 26, 2021
Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.

### Changelog
#### Features

> * Implement Guidance Views. [#4157](https://github.com/mapbox/mapbox-navigation-android/pull/4157)

Reviewer: ```suggestion
- Implemented Guidance Views. See `MapboxJunctionApi` and MapboxJunctionView` for details. [#4157](https://github.com/mapbox/mapbox-navigation-android/pull/4157)
```

> * Implemented Guidance Views. See `MapboxJunctionApi` and MapboxJunctionView` for details. [#4157](https://github.com/mapbox/mapbox-navigation-android/pull/4157)

Reviewer: ```suggestion
* Implemented Guidance Views. See `MapboxJunctionApi` and `MapboxJunctionView` for details. [#4157](https://github.com/mapbox/mapbox-navigation-android/pull/4157)
```

> * Added `SnappingClosures` param to build route request. [#4161](https://github.com/mapbox/mapbox-navigation-android/pull/4161)

Reviewer: ```suggestion
* Added `SnappingClosures` param to route request builder. [#4161](https://github.com/mapbox/mapbox-navigation-android/pull/4161)
```
